frontend/packages/wizard/src/wizard.tsx

```diff
@@ -112,6 +113,7 @@ const Spacer = styled.div<SpacerProps>`
 interface WizardProps {
   heading?: string;
   dataLayout: ManagerLayout;
+  maxWidth?: "xs" | "sm" | "md" | "lg" | "xl" | false;
```

Comment:
can you use the Container type for this field so it stays up to date automatically?